newrelic-alerts-configurator/src/main/java/com/ocadotechnology/newrelic/alertsconfigurator/configuration/DashboardConfiguration.java

```diff
@@ -0,0 +1,45 @@
+package com.ocadotechnology.newrelic.alertsconfigurator.configuration;
+
+import com.ocadotechnology.newrelic.alertsconfigurator.configuration.dashboard.DashboardEditable;
+import com.ocadotechnology.newrelic.alertsconfigurator.configuration.dashboard.DashboardIcon;
+import com.ocadotechnology.newrelic.alertsconfigurator.configuration.dashboard.DashboardVisibility;
+import com.ocadotechnology.newrelic.apiclient.model.dashboards.widget.Widget;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+import java.util.List;
+
+@Getter
+@Builder
+@RequiredArgsConstructor
+public class DashboardConfiguration {
+
+    @NonNull
+    private final String title;
+
+    private final String description;
+
+    private final DashboardIcon icon;
+
+    private final DashboardVisibility visibility;
+
+    private final DashboardEditable editable;
+
+    private final String ownerEmail;
+
+    private final List<Widget> widgets;
```

Comment:
We shouldn't use api DTOs directly. Right now, Widget contains `widgetId` and `accountId` we can set. What are those? Should I be able to set them?
There should be separate configuration DTO to configure Widget, and other objects that are part of the Widget

Reply:
This way, we could use different validation and data types. For instance, Widget in api has a `visualization` which is a String. Maybe in configuration DTO, it could be an enum?